### ISSUE TITLE
Make OutputFile API Fluent

### ISSUE THREAD

**Issue by TylerBrinkley** [daveaglick/Scripty#25]
To support chaining I believe OutputFile should implement a fluent API such that one could do the following.

```
Output.WriteLine("class Foo")
    .WriteLine("{")
    .WriteLine("    int Property => 42;")
    .WriteLine("}");
```

**Comment by daveaglick** [comment]
Right now, `OutputFile` is derived from `TextWriter`. That gives us the benefit of being able to feed an `OutputWriter` into any other API that knows how to work with the native .NET IO classes (which is most of them). While I personally like fluent APIs and think this would be a positive change otherwise, I think that the compatibility with existing IO code is more important. It opens up a lot of doors for letting something else write to the output without having to transform to a string first and manually code up write statements.

Any additional thoughts?

**Comment by daveaglick** [comment]
Ah, I see in the PR that you're hiding the `TextWriter` implementation but it's still there under the hood. Let me take a closer look and get back to you...

**Comment by daveaglick** [comment]
Resolved by #26
